src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String DIVIDER = "____________________________________________________________";
+    private static final Task[] STORED_TASKS = new Task[100];
+    private static int storedTasksCount = 0;
+
     public static void main(String[] args) {
+        String command;
+        Scanner scanner = new Scanner(System.in);
+
+        welcomeMsg();
+        do {
+            command = scanner.nextLine();
+            handleCommand(command);
+        } while(!command.equals("bye"));
+
+        scanner.close();
+    }
+
+    public static void welcomeMsg() {
         String logo = " ____        _        \n"
```

Comment:
Since the Duke logo is unlikely to change while in use, would it perhaps be better to declare this as a constant?